src/api/Alert/index.ts

```diff
@@ -195,9 +208,62 @@ export class AlertAPI extends Request {
   }
 
   /**
-   * Do an action (open, dismiss, confirm, unconfirm) on a match.
+   * Update a comment on a match if the user that triggered the update is the author
+   */
+  public editMatchComment(this: Request<MatchAction>, params: IUpdateMatchCommentParams) {
+    return this.request({
+      errors: [FEATURE_DISABLED, UNAUTHORIZED, DATA_SOURCE_UNAVAILABLE, FORBIDDEN, NOT_FOUND],
+      url: '/:sourceKey/alert/match/comment/:commentId',
+      method: 'PATCH',
+      params: params
+    });
+  }
+
+  /**
+   * Delete a comment on a match if the user that triggered the deletion is the author
+   */
+  public deleteMatchComment(params: IDeleteMatchCommentParams) {
+    return this.request({
+      errors: [FEATURE_DISABLED, UNAUTHORIZED, DATA_SOURCE_UNAVAILABLE, FORBIDDEN, NOT_FOUND],
+      url: '/:sourceKey/alert/match/comment/:commentId',
+      method: 'DELETE',
+      params: params
+    });
+  }
+
+  /**
+   * Get the last created action of a match if any.
+   */
+  public async getLastMatchAction(
+    params: Pick<IGetMatchActionsParams, 'alertId' | 'matchId' | 'sourceKey'>
+  ): Promise<
```

Comment:
What is the objective of this method ?
I feel there might be something wrong here but not sure.
If the objective is to the the last comment on a status change, there might be a bug if the last action is a comment and not a status change.
If the objective is to get the last comment regardless of the action then why have a specific method for that when  `getMatchActions` already exists.

Reply:
Indeed the frontend can just use the `getMatchActions`, this can be negotiated.

@maxime-allex are you ok if we remove `getLastMatchAction` method from the RC? You just need to update this:
```
  public async getLastAction(matchId: number, alertId: number): Promise<MatchAction> {
    const action = await this._restClient.alert.getLastMatchAction({
      matchId: matchId,
      alertId: alertId
    });
    if (action.isSuccess()) {
      return action.body;
    }
  }
  ```
To this:
```
  public async getLastAction(matchId: number, alertId: number): Promise<MatchAction> {
    const action = await this._restClient.alert.getMatchActions({
      matchId: matchId,
      alertId: alertId,
      action: [
        MatchActionType.CONFIRM,
        MatchActionType.DISMISS,
        MatchActionType.OPEN,
        MatchActionType.UNCONFIRM
      ],
      offset: 0,
      limit: 1
    });
    if (action.isSuccess()) {
      return action.body.matchActions[0];
    }
  }
```

Reply:
@maxime-allex are you ok with this ? We are about to start merging everything

Reply:
Of course you can remove it. Seeing the last action is not part of the scope anymore... 🥰